@kiva/kv-components/vue/KvSelect.vue

```diff
@@ -58,21 +58,32 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		/**
+		 * modelValue prop
+		 * */
+		modelValue: {
+			type: [Number, String],
+			default: 0,
+		},
 	},
-	data() {
-		return {
-			mdiChevronDown,
-		};
-	},
-	methods: {
-		onChange(event) {
+	emits: [
+		'change',
+		'update:modelValue',
```

Comment:
this would be `'update:value'`

Reply:
Ahh, thanks for the link. In that case I think we should remove the `value` prop

Reply:
Agreed, the `value` prop should be removed and `modelValue` should replace any usage of it. That will also mean that the model option should changed to:
```
model: {
	prop: 'modelValue',
	event: 'update:modelValue',
},
```

Reply:
Good!
The prop was removed and replaced with modelValue.